### PR TITLE
Added installation manifests and readme for litmus 3.0.0-beta9

### PR DESCRIPTION
**Pull request by Saranya-jena**
## Proposed changes

Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

**Review by SarthakJain26 — APPROVED** [review]
LGTM

**Review by amityt — APPROVED** [review]

**Review by amityt — APPROVED** [review]
LGTM

**Review by Jonsy13 — APPROVED** [review]

File: mkdocs/docs/3.0.0-beta9/litmus-3.0.0-beta9-without-resources.yaml [hunk 1]
```diff
@@ -0,0 +1,778 @@
...
+  namespace: litmus
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: litmus-portal-admin-secret
+  namespace: litmus
+stringData:
+  JWT_SECRET: "litmus-portal@123"
+  DB_USER: "admin"
```
Comment (Jonsy13) [inline mkdocs/docs/3.0.0-beta9/litmus-3.0.0-beta9-without-resources.yaml:434]: This should be `root` i think!

File: chaoscenter/web/nginx/nginx.conf [hunk 2]
```diff
@@ -5,6 +5,11 @@ events {
...
 
 http {
+    map $http_upgrade $connection_upgrade {
+        default upgrade;
+        '' close;
+    }
+
     client_body_temp_path /tmp/client_temp;
     proxy_temp_path       /tmp/proxy_temp_path;
     fastcgi_temp_path     /tmp/fastcgi_temp;
```
Comment (Jonsy13) [inline chaoscenter/web/nginx/nginx.conf:15]: In this file also, change port to `8185` in line 35.

**Pull request merged**